internal/nginx/nginx.go

```diff
@@ -454,15 +444,10 @@ func (nginx *Controller) UpdateConfigVersionFile() {
 	}
 
 	if !nginx.local {
-		w, err := os.Create(tempname)
+		err := createFileAndWrite(tempname, cfg)
 		if err != nil {
-			glog.Fatalf("Failed to open %v: %v", filename, err)
+			glog.Fatalf("%v", err.Error())
```

Comment:
Similar to the main NGINX conf, better to add more context to the error message: "Failed to write version config: %v"